scalameta/tokenizers/shared/src/main/scala/scala/meta/internal/tokenizers/LegacyToken.scala

```diff
@@ -67,6 +67,7 @@ object LegacyToken {
   final val TYPE = 70
   final val FORSOME = 71
   final val DEF = 72
+  final val EXTENSION = 139
```

Comment:
can we move it to below `GIVEN = 76` and assign number 77?
or below `CTXARROW = 138` 

just to keep this list sorted

Reply:
Changed it to 76, good point.